### PR TITLE
Increase the sleep timer in unit tests

### PR DESCRIPTION
**Pull request by David2464**
We have discovered that in order for some tests to pass, they require more time to connect to their desired process or for the processes themselves to start up. In order to give them more time we have increased the sleep timer in the test.csv's. The files affected are within the stp directory and are the wdb, upds, tpvalidation, recovery and subscription.

### CODE REVIEW

File: tests/stp/recovery/recoveryreplayperiod.csv [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 before,0,0,q,stpHandle(`.stplg.init;testlogdb),1,,"Create test stplog test directory"
 before,0,0,q,logdir:1_string stpHandle(`.stplg.dldir),1,,"Get location of log directory"
 before,0,0,q,startproc["rdball"],1,,"Start RDB"
-before,0,0,q,.proc.sys "sleep 1",1,,"Wait for procs to start up"
+before,0,0,q,.proc.sys "sleep 10",1,,"Wait for procs to start up"
 before,0,0,q,rdbHandle:gethandle[`rdball],1,,"Open handle to RDB"
 run,0,0,q,c1:rdbHandle"count trade",1,,"Get initial trade table count"
 run,0,0,q,stpHandle(`.u.upd;`trade;testtrade),1,,"Send trade update to segmented tickerplant"
-run,0,0,q,.proc.sys "sleep 11",1,,"Wait for 10 second period to elapse"
+run,0,0,q,.proc.sys "sleep 15",1,,"Wait for 10 second period to elapse"
```
Comment (jonathonmcmurray) [inline tests/stp/recovery/recoveryreplayperiod.csv:13]: comment is wrong (it already was wrong, but it's even more wrong now)

File: tests/stp/recovery/recoveryreplayperiod.csv [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 true,0,0,q,(c1+10)~c2:rdbHandle"count trade",1,,"Check update has published"
 run,0,0,q,kill9proc["rdball"],1,,"Kill RDB"
 true,0,0,q,not isalive["rdball"],1,,"Make sure it is dead"
 run,0,0,q,startproc["rdball"],1,,"Restart RDB"
-run,0,0,q,.proc.sys "sleep 1",1,,"Wait a sec"
+run,0,0,q,.proc.sys "sleep 10",1,,"Wait a sec"
```
Comment (jonathonmcmurray) [inline tests/stp/recovery/recoveryreplayperiod.csv:18]: comment wrong now

File: tests/stp/wdb/tabular.csv [hunk 3]
```diff
@@ -1,30 +1,31 @@
...
+run,0,0,q,.proc.sys "sleep 8",1,,"Sleep for 8 seconds"
 run,0,0,q,stpHandle:gethandle[`stptabular],1,,"Open handle to segmented tickerplant"
 run,0,0,q,stpHandle(`.stplg.init;testlogdb),1,,"Switch test stplog directory"
 run,0,0,q,logdir:stpHandle(`.stplg.dldir),1,,"Get testlog directory"
 run,0,0,q,"stpHandle @/: `.u.upd ,/: ((`trade;testtrade);(`quote;testquote))",1,,"Send second updates to STP"
 run,0,0,q,startproc each ("wdball";"wdbsymfilt";"wdbtabfilt"),1,,"Start all WDBs"
-run,0,0,q,.proc.sys "sleep 2",1,,"Sleep for 2 seconds"
+run,0,0,q,.proc.sys "sleep 8",1,,"Sleep for 8 seconds"
 run,0,0,q,wdbHandles:`all`sym`tab!gethandle each `wdball`wdbsymfilt`wdbtabfilt,1,,"Open handles to WDBs"
+run,0,0,q,show wdbHandles,1,,"Show wdbHandles"
```
Comment (jonathonmcmurray) [inline tests/stp/wdb/tabular.csv:19]: this was for debugging, shouldn't be in final tests

File: tests/stp/recovery/recoveryreplayperiod.csv (hunk 1 above)
Comment (andrewMorrison95) [reply tests/stp/recovery/recoveryreplayperiod.csv]: rectified

**Review by zanbuchanan — APPROVED** [review]

**Pull request merged**